File: src/main/java/com/skidders/sigma/module/settings/impl/BooleanSetting.java
package com.skidders.sigma.module.settings.impl;

import com.skidders.sigma.module.settings.Setting;

public class BooleanSetting extends Setting<Boolean> {
    public BooleanSetting(String name, String desc, boolean value) {
        super(name, desc, value);
    }
}

File: src/main/kotlin/com/skidders/sigma/module/settings/impl/BooleanSetting.kt
package com.skidders.sigma.module.settings.impl

import com.skidders.sigma.module.settings.Setting
import com.skidders.sigma.screens.clickgui.components.CheckboxComponent

class BooleanSetting : Setting<Boolean> {
    val checkboxComponent: CheckboxComponent = CheckboxComponent()
    constructor(name: String, desc: String, value: Boolean) : super(name, desc, value)
}
